src/flowMC/resource/kernel/MALA.py

```
@@ -169,6 +169,24 @@ def adapt_step_size(
         new_step_size = self.step_size * (1.0 + self.ADAPTATION_RATE * diff)
         return tree_at(lambda k: k.step_size, self, new_step_size)
 
+    def get_effective_dim_profile(self) -> Float[Array, " n_dim"]:
```

Ruff (0.15.13)

[error] Syntax error in forward annotation: Expected an expression

(F722)